### PR TITLE
clarifying device support for deprecated and unsupported

### PR DESCRIPTION
**Pull request by Danny213123**
- clarified deprecated support
- clarified unsupported

### CODE REVIEW

File: docs/reference/system-requirements.rst [hunk 1]
```diff
@@ -59,9 +59,9 @@ GPU is not listed on this table, it's not officially supported by AMD.
 ✅: **Supported** - AMD enables these GPUs in our software distributions for the corresponding
 ROCm product.
 
-⚠️: **Deprecated** - Support will be removed in a future release.
+⚠️: **Deprecated** - Support for deprecated devices will be removed in a future release. While all existing functionalities on these GPUs will be maintained, no new functionalities may be added.
```
Comment (lpaoletti) [inline docs/reference/system-requirements.rst]: Can a device be deprecated?

Comment (lpaoletti) [reply docs/reference/system-requirements.rst]:
```suggestion
⚠️: **Deprecated** - All existing GPU functionality will be maintained, and no new functionality will be added.
```

File: docs/reference/system-requirements.rst [hunk 2]
```diff
@@ -59,9 +59,9 @@ GPU is not listed on this table, it's not officially supported by AMD.
...
 
-❌: **Unsupported** - This configuration is not enabled in our software distributions.
+❌: **Unsupported** - HIP runtime may continue to work but prebuilt ROCm libraries published by AMD no longer support your product and will cause run-time errors.
```
Comment (lpaoletti) [inline docs/reference/system-requirements.rst]:
```suggestion
❌: **Unsupported** - HIP runtime might continue to perform as expected, but prebuilt ROCm libraries published by AMD are no longer supported and will result in runtime errors.
```

File: docs/reference/system-requirements.rst [hunk 3]
```diff
@@ -59,9 +59,9 @@ GPU is not listed on this table, it's not officially supported by AMD.
 ✅: **Supported** - AMD enables these GPUs in our software distributions for the corresponding
```
Comment (peterjunpark) [inline docs/reference/system-requirements.rst]:
```suggestion
✅: **Supported** - Official software distributions of the current ROCm release fully support this hardware. All features and optimizations work as expected.
```

File: docs/reference/system-requirements.rst [hunk 4]
```diff
@@ -59,9 +59,9 @@ GPU is not listed on this table, it's not officially supported by AMD.
...
 ROCm product.
 
-⚠️: **Deprecated** - Support will be removed in a future release.
+⚠️: **Deprecated** - All existing GPU functionality will be maintained, and no new functionality will be added.
 
-❌: **Unsupported** - This configuration is not enabled in our software distributions.
+❌: **Unsupported** - HIP runtime might continue to perform as expected, but prebuilt ROCm libraries published by AMD are no longer supported and will result in runtime errors.
```
Comment (peterjunpark) [inline docs/reference/system-requirements.rst]:
```suggestion
❌: **Unsupported** - The current ROCm release does not support this hardware. The HIP runtime can continue to work, but prebuilt ROCm libraries are not officially supported and will cause runtime errors. Consider upgrading to fully supported hardware.
```

File: docs/reference/system-requirements.rst [hunk 5]
```diff
@@ -59,9 +59,9 @@ GPU is not listed on this table, it's not officially supported by AMD.
 ✅: **Supported** - AMD enables these GPUs in our software distributions for the corresponding
 ROCm product.
 
-⚠️: **Deprecated** - Support will be removed in a future release.
+⚠️: **Deprecated** - All existing GPU functionality will be maintained, and no new functionality will be added.
```
Comment (peterjunpark) [inline docs/reference/system-requirements.rst]:
```suggestion
⚠️: **Deprecated** - The current ROCm release has limited support for this hardware. Existing features and capabilities are maintained, but no new features or optimizations will be added. A future ROCm release will remove support. Consider upgrading to fully supported hardware.
```

**Comment by peterjunpark** [comment]
I felt like the descriptions could use a little more clarification. Left my suggestions

**Comment by Danny213123** [comment]
I have made some changes, please take a look

**Review by peterjunpark — APPROVED** [review]
lgtm

File: docs/reference/system-requirements.rst [hunk 6]
```diff
@@ -56,12 +56,11 @@ GPU is not listed on this table, it's not officially supported by AMD.
       "AMD Radeon RX 7900 GRE", "RDNA3", "gfx1100", "✅"
       "AMD Radeon VII", "GCN5.1", "gfx906", "⚠️"
 
-✅: **Supported** - AMD enables these GPUs in our software distributions for the corresponding
-ROCm product.
+✅: **Supported** - Official software distributions of the current ROCm release fully enables this hardware.
```
Comment (peterjunpark) [inline docs/reference/system-requirements.rst:59]:
```suggestion
✅: **Supported** - Official software distributions of the current ROCm release fully support this hardware.
```
"enables" sounds weird to me. "supports" sounds more natural to me in this context, but not a blocker.

Also, we should use the appropriate verb tense here. THanks

**Review by yhuiYH — APPROVED** [review]
The more detailed explanations are a move in the positive direction IMO.  This LGTM.

**Pull request merged**